src/config.ts

```diff
@@ -80,6 +82,12 @@ const validators = {
   REACT_APP_PENDO_VISITOR_ID_PREFIX: envalid.str({ default: 'test_visitor' }),
 }
 
+// @TODO: We may want to move internal constants out of config and into a separate file
```

Comment:
let's action this todo while we're here. 

while i understand the necessity for moving this constant (avoid a circular dep) and the rationale for it living here (it's configish), i would much rather this particular thing (`DEFAULT_HISTORY_TIMEFRAME`) and similar live in a `constants.ts` file or similar.

the `config.ts` file has a singular concern - parse, validate, and expose typed config from the `.env` file, this changes that pattern and creates a temptation and slippery stuff for other non `.env` things to end up in here.

Reply:
Actioned in https://github.com/shapeshift/web/pull/2209/commits/b158b6752359d284385cbd90d97656d38bcd1de2